packages/marketing-modal/src/blobs.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import { palette } from '@leafygreen-ui/palette';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { BlobPosition } from '.';
+
+type SvgBlobProperties = 'viewBox' | 'path' | 'fill' | 'styles';
+
+const svgBlobPaths: Record<BlobPosition, Record<SvgBlobProperties, string>> = {
+  [BlobPosition.TopLeft]: {
+    viewBox: '0 0 265 501',
+    path:
+      'M-10.0239 -12.0209L154.458 -12.0208C215.515 -12.0208 265 52.2446 265 131.538C265 210.041 215.934 273.615 155.523 273.467L98.8817 273.318C68.886 273.269 44.5623 304.808 44.5623 343.714L44.5623 346.186C44.5623 385.091 20.2765 416.581 -9.64308 416.581L-10.709 416.581C-40.7047 416.581 -64.9905 448.219 -64.9144 487.174C-64.8382 526.129 -89.1241 557.767 -119.12 557.767L-120.604 557.767C-150.524 557.767 -174.772 526.277 -174.81 487.421L-175 201.242C-175.038 162.336 -150.752 130.797 -120.795 130.797L-118.435 130.797C-88.477 130.797 -64.2292 99.2574 -64.2292 60.4015L-64.2292 58.3251C-64.2292 19.5186 -39.9815 -12.0209 -10.0239 -12.0209Z',
+    fill: `${palette.purple.light3}`,
```

Comment:
Should the blob color be customizable?

Reply:
For now no, but it will probably need to change when `darkMode` is updated